### PR TITLE
Fix the problem that the input box on android platform cannot display the content of soft keyboard immediately

### PR DESCRIPTION
**Pull request by bofeng-song**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by minggo** [comment]
@bofeng-song any issue for this PR?

File: native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java [hunk 1]
```diff
@@ -466,7 +466,7 @@ public void run() {
      Native functions invoked by UI.
      **************************************************************************************/
     private void onKeyboardInput(String text) {
-        CocosHelper.runOnGameThreadAtForeground(new Runnable() {
+        CocosHelper.runOnGameThread(new Runnable() {
```
Comment (minggo) [inline native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java:469]: What's the difference between these two functions?

Comment (bofeng-song) [reply native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java:469]: When an activity is in the foreground (activity is in the resume state), two interfaces can be called, but when the activity is in the pause state, only runOnGameThread interface can be called

Comment (minggo) [reply native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java:469]: Got it. But if the activity in pause state, why `onKeyboardInput ` can be invoked?

Comment (bofeng-song) [reply native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java:469]: CocosEditBoxActivity is a transparent activity, game can be show normally, and game's render loop still running

Comment (minggo) [reply native/cocos/platform/android/java/src/com/cocos/lib/CocosEditBoxActivity.java:469]: Got it.

**Review by minggo — APPROVED** [review]

**Pull request merged**